src/anomalib/models/reverse_distillation/torch_model.py

```diff
@@ -45,12 +45,7 @@ def __init__(
         self.bottleneck = get_bottleneck_layer(backbone)
         self.decoder = get_decoder(backbone)
 
-        if self.tiler:
-            image_size = (self.tiler.tile_size_h, self.tiler.tile_size_w)
-        else:
-            image_size = input_size
-
-        self.anomaly_map_generator = AnomalyMapGenerator(image_size=image_size, mode=anomaly_map_mode)
+        self.anomaly_map_generator = AnomalyMapGenerator(image_size=input_size, mode=anomaly_map_mode)
```

Comment:
A reference for future, but not it might make more sense to rename `image_size` argument to `input_size` in `AnomalyMapGenerator`. As I said, it is not related to this PR, though